src/asset-swapper/utils/market_operation_utils/constants.ts

```diff
@@ -1431,6 +1436,15 @@ export const KYBER_DMM_ROUTER_BY_CHAIN_ID = valueByChainId<string>(
     NULL_ADDRESS,
 );
 
+export const KYBER_ELASTIC_CONFIG_BY_CHAIN_ID = valueByChainId(
+    {},
```

Comment:
This map will default to a valid quoter, factory, and router addresses for every chain. I think this will cause sampler to fail for chains other than Mainnet and Polygon.

Reply:
This is intended to mean the addresses are the same for every chain. The source shouldn't be sampled if it is not listed in the constant SELL_SOURCE_FILTER_BY_CHAIN_ID. I'll give it a test.

Reply:
I ran simbot for CELO chain, and it runs fine. The sources not listed in the source filters shouldn't be sampled, so we can control which samplers are "active" for each chain.

Reply:
Got it, sounds good to me.